integreat_compass/cms/apps.py

```diff
@@ -19,4 +19,4 @@ def ready(self):
         # pylint: disable-next=import-outside-toplevel
         from django.contrib.auth.models import Group
 
-        Group.__str__ = lambda self: _(self.name)
+        Group.__str__ = lambda self: self.name
```

Comment:
If you remove the translation, you can also remove the whole line.
The default string representation is already the name:
https://github.com/django/django/blob/c3862735cd8c268e99fb8d54c3955aacc4f2dc25/django/contrib/auth/models.py#L129-L130

Reply:
Alright. Just for an explanation: This translation prevented the creation of new groups in the backend, throwing `TypeError: __str__ returned non-string (type __proxy__)`. See also: https://stackoverflow.com/questions/51959209/django-error-str-returned-non-string-type-proxy

Using the non-lazy translate version gave a different set of errors, so I just opted to removing it.

Reply:
`AppRegistryNotReady` - but yes, with the lazy for the verbose_name it appears to be working again 🤔